beets/dbcore/query.py

```diff
@@ -140,14 +148,17 @@ class RegexpQuery(StringFieldQuery):
     """A query that matches a regular expression in a specific item
     field.
     """
-    @classmethod
-    def string_match(cls, pattern, value):
+    def __init__(self, field, pattern, false=True):
```

Comment:
It might be nice to add to the docstring when a constructor is allowed to raise an `InvalidQueryError`. I can help write these post-merge.